client/templates/users/user_profile.js

```diff
@@ -13,5 +13,9 @@ Template.userProfile.helpers({
 
   memberOfProjects: function() {
     return Projects.find({ "members.userId": Meteor.userId() }, { "userId": 1, "username": 1});
+  },
+
+  isMemberOf: function() {
+    return Projects.find({ "members.userId": Meteor.userId() }, { "userId": 1, "username": 1}).count() > 0;
```

Comment:
Derp, Told you I can't program

On Thu, 7 Apr 2016 16:11 Benny Zlotnik, <notifications@github.com> wrote:

> In client/templates/users/user_profile.js
> <https://github.com/PUMATeam/puma/pull/69#discussion_r58867864>:
>
> > @@ -13,5 +13,9 @@ Template.userProfile.helpers({
> >
> >    memberOfProjects: function() {
> >      return Projects.find({ "members.userId": Meteor.userId() }, { "userId": 1, "username": 1});
> > +  },
> > +
> > +  isMemberOf: function() {
> > +    return Projects.find({ "members.userId": Meteor.userId() }, { "userId": 1, "username": 1}).count() > 0;
>
> you can reuse the other function
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/PUMATeam/puma/pull/69/files/4049f6fc60cb4b0eb945045f2df22e58f44e83e8#r58867864>
>